rholang/src/main/scala/coop/rchain/rholang/interpreter/implicits.scala

```diff
@@ -200,16 +200,86 @@ object implicits {
         that.freeCount + p.freeCount,
         that.locallyFree | p.locallyFree
       )
+
+    def wildcard: Boolean =
+      p.sends.map(send => SendLocallyFree.wildcard(send)).exists(identity) || p.evals
+        .map(eval => EvalLocallyFree.wildcard(eval))
+        .exists(identity) || p.exprs
+        .map(expr => ExprLocallyFree.wildcard(expr))
+        .exists(identity) || p.receives
+        .map(receive => ReceiveLocallyFree.wildcard(receive))
+        .exists(identity) || p.matches
+        .map(_match => MatchLocallyFree.wildcard(_match))
+        .exists(identity)
   }
 
   implicit def fromPar[T](p: T)(implicit toPar: T => Par): Option[Par] = Some(p)
 
   implicit val SendLocallyFree: HasLocallyFree[Send] = new HasLocallyFree[Send] {
+    def wildcard(s: Send) =
+      ChannelLocallyFree.wildcard(s.chan.get) || s.data.map(_.wildcard).exists(identity)
```

Comment:
Same here. Any of the protos that have freeCount explicitly should have wildcard there too.

Reply:
Done